libraries/inspec.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+class Audit
+  class Resource
+    class ChefInspec < Chef::Resource
+      resource_name :inspec
+
+      property :version, String, default: 'latest'
+
+      default_action :install
+
+      # installs inspec if required
+      action :install do
+        converge_by 'install/update inspec' do
+          chef_gem 'inspec' do
+            version version if version != 'latest'
```

Comment:
@chris-rock I get an WARN on line 15 `version` 

Recipe: audit::_inspec
``` 
 * inspec[inspec] action install[2016-08-25T13:28:53+00:00] WARN: /var/chef/cache/cookbooks/audit/libraries/inspec.rb:15:in `block (3 levels) in <class:ChefInspec>': property version is declared in both chef_gem[inspec] and inspec[inspec] action :install. Use new_resource.version instead. At /var/chef/cache/cookbooks/audit/libraries/inspec.rb:15:in `block (3 levels) in <class:ChefInspec>'
[2016-08-25T13:28:53+00:00] WARN: /var/chef/cache/cookbooks/audit/libraries/inspec.rb:15:in `block (3 levels) in <class:ChefInspec>': property version is declared in both chef_gem[inspec] and inspec[inspec] action :install. Use new_resource.version instead. At /var/chef/cache/cookbooks/audit/libraries/inspec.rb:15:in `block (3 levels) in <class:ChefInspec>'
```
This doc seems to support the WARN message above:
https://docs.chef.io/custom_resources.html#new-resource-property

When I test changing line 15 to the following I no longer get the WARNing.
`version new_resource.version if new_resource.version != 'latest'`